signac/contrib/__init__.py

```diff
@@ -25,7 +25,6 @@
 
 logger = logging.getLogger(__name__)
 
-
```

Comment:
Is this something the linter complained about? Otherwise, please revert this change.